test/general/key.js

```diff
@@ -2305,7 +2305,7 @@ VYGdb3eNlV8CfoEC
   });
 
   it('require_uid_self_cert=true: cannot use pubkey w/o self certs', async function() {
-    let k = openpgp.key.readArmored(no_self_cert.one_user_no_self_cert).keys[0];
+    let {keys: [k]} = await openpgp.key.readArmored(no_self_cert.one_user_no_self_cert);
```

Comment:
Huh, that's much more elegant than `(await ...).keys[0]`. Why didn't I think of that 🙃

Reply:
I got that particular piece of ES6 from https://www.youtube.com/watch?v=AfWYO8t7ed4 a few months back after converting my project to TypeScript. Useful!